plugin.c

```diff
@@ -210,7 +210,7 @@ const char *hts_plugin_path(void) {
     }
 
     static char s_path[1024];
-    sprintf(s_path, "%.1023s", ks.s ? ks.s : "");
+    snprintf(s_path, sizeof(s_path), "%.1023s", ks.s ? ks.s : "");
```

Comment:
Again we may as well change this to `%s` now we don't need the length protection.

Reply:
Updated.